pkgs/desktops/gnome/extensions/extensionRenames.nix

```diff
@@ -26,4 +26,42 @@
 
   # That extension is broken because of https://github.com/NixOS/nixpkgs/issues/118612
   #"flypie@schneegans.github.com" = null;
+
+  # These are overrides for the manually packaged ones.
+  "appindicatorsupport@rgcjonas.gmail.com" = "appindicator";
```

Comment:
Maybe we could even use `passthru.extensionSlug` to make this even more consistent.

Reply:
You mean for the value of the attributes? This would result in some extensions being renamed (at least their attribute in `nixpkgs`), requiring an alias. As I learned, renaming extensions is a rather tricky topic (since the `extensionSlug` may contain errors), so I thought I'd leave them as they are now and do that some time later (the pnames need fixing as well, for Repology consistency).

Btw these lines cause some of the automatically packaged extensions to be renamed too. I consider this a good thing, because it previously was weird having the automatically and manually packaged extensions available under different attribute names.

Reply:
I meant that we would add `passthru.extensionSlug` containing manually selected `pname` to the manually packaged extensions and then `mapReadableNames` would use that without the need to define aliases here. But maybe having the names for manually packaged extensions is more sensible approach. Not sure.

Reply:
Please reiterate. What is exactly your problem here and what do suggest?

Reply:
The “issue” is that we essentially have two different ways of setting readable names. For automatically packaged extension, it comes from the `passthru.extensionSlug` field, while for the manually packaged, we set the name here. I would suggest unifying it.

But I would not block on this. We can resolve it later if we decide it is a reasonable change.

Reply:
Oh, now I understand your concern. I pushed a commit that kind of does what you suggested: let me know what you think of it (and good luck trying to catch any potential mishaps/typos). I'll try to clean up the history afterwards.